test/runtests.jl

```diff
@@ -1,5 +1,8 @@
 using BlockDiagonals
+using ChainRulesCore
+using ChainRulesTestUtils
 using Documenter
+using FiniteDifferences
```

Comment:
```suggestion
using FiniteDifferences: FiniteDifferences  # we only have this to overload `to_vec`
```

When I look at this i keep thinking "why are we usign FIniteDifferences directly".
So maybe a comment to remind why